### DISCUSSION TITLE
about

### DISCUSSION THREAD

**Discussion by tw93** [tw93/tw93.github.io#106] [General]
https://tw93.fun/about.html

 Hi，我是 Tw93，是一个前端工程师，空闲也折腾 iOS / macOS ，喜欢开源分享交流。

**Comment by sbccznfy** [comment]
大佬有兴趣把weekly同步到这个博客吗，或者单开一个专栏之类的，很喜欢您的博客😁

**Comment by tw93** [comment]
> @sbccznfy
> 大佬有兴趣把weekly同步到这个博客吗，或者单开一个专栏之类的，很喜欢您的博客😁

哈哈 终于来了  http://weekly.tw93.fun/

**Comment by harleyszhang** [comment]
大佬方便问下你这个博客网站主题是哪里，用的什么框架呢

**Reply by tw93** [reply]
可以点击底部链接，就是仓库地址了

**Comment by honglonglong** [comment]
您好请问有rss或newsletter吗

**Reply by tw93** [reply]
有的 你可以用pc切到首页，顶部菜单就是

**Reply by honglonglong** [reply]
嗯看到了，谢谢

**Comment by hahahaDamon** [comment]
想问下，潮流周刊和博客的区别是什么呢？~ 潮流周刊主要是资讯？博客主要是感想？

**Reply by tw93** [reply]
可以这样理解 博客是创作，周刊是收集学习

**Comment by link-hao** [comment]
偶然发现作者的的博客，看头像原来之前在twitter已经关注了，果然厉害的总是那么一波少数人

**Comment by Yiju-Li** [comment]
不愧是前端工程师！大佬的所有产品设计得都非常的赏心悦目！

**Reply by tw93** [reply]
谢谢喜欢

**Comment by yyt520** [comment]
大佬 有没有类似公众号之类的呢，我想每次您发了文章我都可以收到通知这样的，读您的文章感觉超棒

**Reply by tw93** [reply]
可以试试用 RSS，订阅这个地址即可 https://tw93.fun/feed.xml
推荐 netnewswire 这个软件

**Reply by Kirin-lee-git** [reply]
在Follow上订阅了，是第一个follower

**Comment by Buji-Calm** [comment]


**Reply by tw93** [reply]
github.com/lxgw/lxgwwenkai-screen
试试这个

**Reply by KuringM** [reply]
现在还是这个字体吗？🤔

**Comment by fantingsheng** [comment]
希望与你交换友链 https://spacexcode.com/

**Reply by tw93** [reply]
抱歉，暂时博客由于设计缘故先不会加入友联功能
ps 你的 分享硬核且有趣的前端编程知识  做得挺好的～

**Comment by shixiaoquan** [comment]
test

**Comment by xiaochengsky** [comment]
这个模板超级好，谢谢大佬！

**Comment by hiXgb** [comment]
Mole挺好用的，可以加个联系方式不，哈哈，也小小喂了下🐱

**Reply by tw93** [reply]
感谢感谢，我有电报群，你可以在妙言里面找找看

**Comment by zzzfoxz** [comment]
大佬博客里的框图是用什么软件画的，想借鉴一下🥰

**Reply by tw93** [reply]
现在是 claude 画svg，之前的手画keynote

**Comment by zhaojc** [comment]
中国人厉害哈哥给你点赞

**Comment by songlindong** [comment]
怎么加个联系方式
